### DISCUSSION TITLE
Option for Lightroom Presets to use Filenames instead of LUT name

### DISCUSSION THREAD

**Discussion by elliotttate** [videovillage/community#25] [Ideas]
I LOVE the ability to export to Lightroom Presets. I've found that Lightroom is one of the best ways to preview a lot of different LUTS very easily and also a great way to color grade.

Unfortunately, a lot of my LUTS have some generic name inside the .cube file like: TITLE "Generate by Resolve"

The actual .cube file names are unique though. Is there any way you can add the option to create the Lightroom preset by Filename instead of LUT name (label within the .cube file?) Thanks so much for all your hard work!!

**Comment by gregcotten** [comment]
Hi Elliot,

Sounds doable! I'll squeeze this in an update soon.
